pkg/graveler/graveler_test.go

```diff
@@ -2292,3 +2292,182 @@ func TestGravelerCreateCommitRecord(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestGraveler_Revert(t *testing.T) {
+	type fields struct {
+		CommittedManager *testutil.CommittedFake
+		RefManager       *testutil.RefsFake
+	}
+	type args struct {
+		ctx          context.Context
```

Comment:
**Go language note.**  This is never used, and probably never should be.  Having it here means encourages passing a `nil` Context, which you should actually never do!  The [`context` godoc](https://pkg.go.dev/context) says:
> Do not pass a nil [Context](https://pkg.go.dev/context#Context), even if a function permits it. Pass [context.TODO](https://pkg.go.dev/context#TODO) if you are unsure about which Context to use.

My recommendation: remove this field, create `ctx := context.TODO()` at the top of the test, and just pass that `ctx` to Revert.